.github/workflows/release.yml

```
@@ -0,0 +1,341 @@
+    - name: Install packaging tools
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y build-essential devscripts debhelper rpm
+    
+    - name: Create DEB package
+      run: |
+        mkdir -p packaging/deb/vibe-ensemble-mcp/DEBIAN
+        mkdir -p packaging/deb/vibe-ensemble-mcp/usr/local/bin
+        mkdir -p packaging/deb/vibe-ensemble-mcp/etc/vibe-ensemble
+        mkdir -p packaging/deb/vibe-ensemble-mcp/usr/lib/systemd/system
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)